CREDITS

```diff
@@ -42,8 +42,6 @@ plugins/micromega
   of Cambridge, 1998)
 plugins/nsatz
   developed by Loïc Pottier (INRIA-Marelle, 2009-2011)
-plugins/omega
-  developed by Pierre Crégut (France Telecom R&D, 1996)
```

Comment:
Careful, `plugins/omega` actually means `plugins/omega` and `theories/omega`. Remove this part only if you are sure there is no more code from Pierre in`theories/omega`.